spec/beaker/hypervisor/openstack_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'fog'
+
+module Beaker
+  describe OpenStack do
+
+    let(:options) { make_opts.merge({'logger' => double().as_null_object}) }
+
+    let(:openstack) {
+      stub_const("Fog::Compute", MockComputeClient)
+      stub_const("Fog::Network", MockNetworkClient)
```

Comment:
@kevpl I'am not experienced in ruby and rspec, but I don't know how to test such cases without mock when you need response and execute one more method of returned object.

For example:
```
vm = @compute_client.servers.create(options)
```